wezterm-gui/src/termwindow/selection.rs

```diff
@@ -58,29 +63,42 @@ impl super::TermWindow {
     ) {
         self.selection(pane.pane_id()).seqno = pane.get_current_seqno();
         let mode = mode.unwrap_or(SelectionMode::Cell);
-        let (x, y) = match self.pane_state(pane.pane_id()).mouse_terminal_coords {
-            Some(coords) => coords,
+        let (x, y, position) = match self.pane_state(pane.pane_id()).mouse_terminal_coords {
+            Some(coords) => (coords.0.column, coords.1, coords.0),
```

Comment:
More of a nit, but: `(coords.0.column, coords.1, coords.0)` feels awkward and looks a bit opaque.

I feel like it would read better here if this was:

```rust
let (position, y) = match self.pane_state(pane.pane_id()).mouse_terminal_coords {
   Some(coords) => coords,
   None => return,
};
```

and then the code below would use `position.column` instead of `x`

What do you think?